scripts/sync.py

```diff
@@ -128,14 +135,25 @@ def main():
 
     # run downloader once, or continuously
     if args.continuous:
+        dates_email_sent = []
         while True:
             do(args, Lochness)
+
+            # daily email
+            if args.daily_summary and date.today() not in dates_email_sent:
+                send_out_daily_updates(Lochness)
+                dates_email_sent.append(date.today())
+
             poll_interval = int(Lochness['poll_interval'])
             logger.info('sleeping for {0} seconds'.format(poll_interval))
             time.sleep(Lochness['poll_interval'])
     else:
         do(args, Lochness)
 
+        # email
+        if args.daily_summary:
+            send_out_daily_updates(Lochness)
```

Comment:
TODO:
Discuss `send_out_daily_updates()` frequency with Kevin.

Reply:
I would vote for an email everyday.

Reply:
Okay, I understand the logic now.